### DISCUSSION TITLE
Anyone have dbdev login working with Windows (powershell specifically)

### DISCUSSION THREAD

**Discussion by GaryAustin1** [supabase/dbdev#145] [Q&A]
I get the token request with a blinking curser.   Nothing pastes in and can't type anything.  Tried pasting and return but not sure if token is even being entered.  Tried in IDE terminal and Windows command prompt.

Result is this error:
```
Please paste the token found on database.dev: 
Error: error parsing toml from credentials file

Caused by:
    TOML parse error at line 3, column 383
      |
    3 | d_8ad48b3a4ced4860b150c7cc02cac01dtGGfBlDBzOv2t7h6NoYvkALHyCL7dbd_8ad48b3a4ced4860b150c7cc02cac01dtGGfBlDBzOv2t7h6NoYvkALHyCL7dbd_8ad48b3a4ced4860b150c7cc02cac01dtGGfBlDBzOv2t7h6NoYvkALHyCL7dbd_8ad48b3a4ced4860b150c7cc02
cac01dtGGfBlDBzOv2t7h6NoYvkALHyCL7dbd_8ad48b3a4ced4860b150c7cc02cac01dtGGfBlDBzOv2t7h6NoYvkALHyCL7dbd_8ad48b3a4ced4860b150c7cc02cac01dtGGfBlDBzOv2t7h6NoYvkALHyCL7"
      |
                                                                                                                                                                  ^
    expected `.`, `=`
```

**Comment by imor** [comment]
@GaryAustin1 the token is not supposed to show when it is pasted. Also looks like your credentials file at `C:\users\<your username>
.dbdev\credentials.toml` can't be parsed. Can you share which version of dbdev CLI are you using (run `dbdev --version` to find out). Can you try deleting the toml file and trying again.

**Reply by GaryAustin1** [reply]
Thanks for the reply!
 'dbdev' (0.1.5) is already installed.
 I had assumed it might not display but pasting with 3 different methods (paste has correct value) and hitting return also gets the error.
 I'll mess with the toml file after breakfast although at one point I uninstalled dbdev and started over.

**Reply by GaryAustin1** [reply]
deleting the credentials.toml did it.  I assume I messed up the first time on token entry and then it does not take a new token after that....
